packages/documentation/copy/en/project-config/tsconfig.json.md

```diff
@@ -107,6 +107,25 @@ This lets your `tsconfig.json` focus on the unique choices for your project, and
 
 The `"compilerOptions"` property can be omitted, in which case the compiler's defaults are used. See our full list of supported [Compiler Options](/tsconfig).
 
+## JSConfig Defaults
```

Comment:
This doesn't feel like the right place to be defining these defaults, given the existing contents of the page (very short, minimal examples, links to other pages). This isn't even a page I think people visit, compared to https://www.typescriptlang.org/tsconfig.

Maybe it'd be clearer to put these into the actual tsconfig page per-option as part of the defaults?

Reply:
I think I agree with @MatthewPinheiro's reasoning. https://www.typescriptlang.org/tsconfig is more of a reference page, with very specific, per-option documentation, while https://www.typescriptlang.org/docs/handbook/tsconfig-json.html is structured more like an article, explaining the "why"s and "how"s of the feature in broader strokes.

I think this section makes sense here, but also agree with @jakebailey that we should also indicate where per-option defaults differ between `tsconfig.json` and `jsconfig.json`.

Reply:
Just checking in to ask what next steps from here would be? Since @rbuckton agrees with adding to both pages, should I add the per-option JSConfig defaults to the reference to this PR as well?

Reply:
Sure; each of the options has its own `md` file, and if any of them differ, you should be able to note that in the text there at the bottom or something.

Feasibly, the TS code that generates the default column could be changed, but that may get a little hairy for info that few people need.